README.md

```diff
@@ -1 +1,24 @@
-# .github
\ No newline at end of file
+# Akka - Build highly concurrent, distributed, and resilient message-driven applications on the JVM 
+
+Welcome to the Akka family of projects!
+
+[Akka](https://akka.io/)
+
+[Akka Management](https://developer.lightbend.com/docs/akka-management/current/)
+
+[Akka gRPC](https://developer.lightbend.com/docs/akka-grpc/current/)
+
+[Akka Http](https://doc.akka.io/docs/akka-http/current/)
```

Comment:
since there seems to be no particular order I think Akka Http deserves to be number 2.